src/sas/sasgui/plottools/toolbar.py

```diff
@@ -64,11 +64,20 @@ def _init_toolbar(self):
         self.AddSimpleTool(self._NTB2_FORWARD, _load_bitmap('forward.png'),
                            'Forward', 'Forward navigation view')
         # todo: get new bitmap
-        self.AddCheckTool(self._NTB2_PAN, _load_bitmap('move.png'),
-                          shortHelp='Pan',
-                          longHelp='Pan with left, zoom with right')
-        self.AddCheckTool(self._NTB2_ZOOM, _load_bitmap('zoom_to_rect.png'),
-                          shortHelp='Zoom', longHelp='Zoom to rectangle')
+        # todo: update with code from matplotlib/backends/backend_wx.py
+        is_phoenix = 'phoenix' in wx.PlatformInfo
+        if is_phoenix: # wx phoenix >= 4.0.0b2
```

Comment:
Nit pick: Unless you plan on using is_phoenix again.
`if 'phoenix' in wx.PlatformInfo`